src/common/transformations/src/transformations/common_optimizations/fuse_rotary_positional_embeddings.cpp

```diff
@@ -575,7 +575,8 @@ ov::pass::RoPEFusionChatGLM::RoPEFusionChatGLM(int split_output_id, const bool s
         auto const_target_shape_2 = makeConst({batch, 1, seq_len, ndims / 2, 2});
 
         // Slice cos_sin_cache to support 2-dimentional RoPE
-        auto ScatterUpdate = makePattern<opset3::ScatterUpdate>({{0, 0}, {1}, seq_length, {0}}, {});
+        auto zero_seqlen = makePattern<opset3::Concat>({{0}, seq_length}, {{"axis", 0}});
```

Comment:
can we extend RoPEFusionChatGLM tests?
probably, it will be enough to apply "symbol_optimization" to the existing test
https://github.com/openvinotoolkit/openvino/blob/25c602597388a91e94ab02dd34a8e1a98ce2a067/src/common/transformations/tests/common_optimizations/fuse_rotary_positional_embeddings.cpp#L481

Reply:
I added SymbolOptimizations to the tests -- lets see how it goes